### PR TITLE
Prepare stable release

### PR DESCRIPTION
**Pull request by alexsomai**
Prepare for stable release version

CC @oleg-nenashev @martinda

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -77,7 +77,7 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>run-selector</artifactId>
-            <version>1.0-beta-2</version>
+            <version>1.0.0</version>
```
Comment (alexsomai) [inline pom.xml]: The current build will fail until I release the stable Run Selector

File: pom.xml [hunk 2]
```diff
@@ -9,7 +9,7 @@
     </parent>
 
     <artifactId>external-workspace-manager</artifactId>
-    <version>1.0-beta-2-SNAPSHOT</version>
+    <version>1.0.0-SNAPSHOT</version>
```
Comment (alexsomai) [inline pom.xml]: I'd choose 1.0.0 for the stable release version

File: README.md [hunk 3]
```diff
@@ -21,7 +21,7 @@ It eliminates the need to copy, archive or move files.
 
 ## Current status
 
-Currently the plugin is under development. See the links to get more info about the current status.
+Version 1.0.0 has been released!
```
Comment (alexsomai) [inline README.md]: Not very sure if this is the best message for the `Current status` section.
Maybe I should remove this section?

Comment (martinda) [inline README.md]: I have seen some projects make use of a version badge. I have no idea how to do that myself.

Comment (alexsomai) [inline README.md]: Adding a badge seems like a good idea.
It shouldn't be that difficult, I've found some resources on Google, but I think that firstly I have to perform the stable release for that to work
https://badge.fury.io/

**Pull request merged**